app/src/main/java/hous/release/android/presentation/profile/ProfileFragment.kt

```diff
@@ -104,6 +123,28 @@ class ProfileFragment : BindingFragment<FragmentProfileBinding>(R.layout.fragmen
                     )
                 }
             }
+            binding.tvProfileBirthday.text = profile.birthday?.substring(5..9)
+            if (profile.introduction.isNullOrEmpty()) {
```

Comment:
```suggestion
            if (profile.introduction.isNullOrBlank()) {
```
introduction에 `" "` 같은 blank가 들어갈 수 있으니 `isNullOrBlank` 추천합니다!!